registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -82,6 +82,7 @@
     "sha256:df8dca5656e3440e6b37f4a2de44a35c4ba20a3b48a0294703f6785365c9f6f5": ["v1.1.5"]
     "sha256:763eac18e8b16f89a8e36f19b052d120231f6ef73f5bbab86457c4d83baa1ce3": ["v1.1.6"]
     "sha256:936dc17db44e0ade12ce57e6f0fff8d258bb847781befbd5d3cec6d21d838291": ["v1.10.0-beta.0"]
+    "sha256:fe4a8024d4ebd86c630331bb8df8ff4e21cc5e1eeeee4112aa278c19b9a64135": ["v1.10.0-beta.1"]
```

Comment:
`fe4a8024d4eb` matches the staging repo: https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:fe4a8024d4ebd86c630331bb8df8ff4e21cc5e1eeeee4112aa278c19b9a64135/details?inv=1&invt=Abs_9w 👍🏻